tempest/api/workloadmgr/workload_reassign/test_workload_reassign_same_project_same_user.py

```diff
@@ -0,0 +1,170 @@
+import os
+import sys
+
+from oslo_log import log as logging
+
+from tempest import config
+from tempest import reporting
+from tempest import tvaultconf
+from tempest.api.workloadmgr import base
+from tempest.lib import decorators
+
+sys.path.append(os.getcwd())
+
+LOG = logging.getLogger(__name__)
+CONF = config.CONF
+
+
+class WorkloadTest(base.BaseWorkloadmgrTest):
+
+    credentials = ['primary']
+
+    @classmethod
+    def setup_clients(cls):
+        super(WorkloadTest, cls).setup_clients()
+        reporting.add_test_script(str(__name__))
```

Comment:
Remove this line, if u are explicitly adding the test script again below.

Reply:
This one is removed.